index.rst

```diff
@@ -6,24 +6,474 @@
+
+::
+
+  touch .config/flake8
+  emacs .config/flake8
+
+
+Then copy-paste the following into the opened config file:
+
+::
+
+  [flake8]
+  max-line-length = 99
+  ignore = E133, E226, E228, E266, N802, N803, N806, N812, N813, N815, N816, W503
+
+Use ``x-s`` then ``x-c`` to save and exit emacs.
+
+
+While developing a notebook
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+While developing a notebook, have the following 'magic' commands as the first code cell:
+
+::
+
+  %load_ext pycodestyle_magic
+  %flake8_on
+  import logging
+  logging.getLogger("flake8").setLevel(logging.FATAL)
+
+Whenever you execute a cell, it will use ``flake8`` to check for adherence to the ``PEP8`` coding style guide, and report violations.
+Fix them as you go.
+Once you're done with the entire notebook you can remove that cell with the magic commands. 
+
+
+When the notebook is complete
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When the notebook is complete, from the command line in the notebook's directory execute:
+
+::
+
+  flake8-nb notebook_name.ipynb
+
+
+This will give you a final check of any violations with ``PEP8``.
+This will catch things that can be missed line-by-line, such as packages that are imported but never used.  
+
+
+Git branch, merge, and review policy for ``tutorial-notebooks``
+===============================================================
+
+The following applies when creating or updating notebooks in the ``tutorial-notebooks`` repository of the ``rubin-dp0`` GitHub Org.
+
+Branch
+------
+
+Develop new notebooks, or update existing ones, in a new branch.
+This branch should be named for the corresponding Jira ticket (e.g., "tickets/PREOPS-12345").
+The new branch should be made from ``main``, *not* from ``prod``.
+
+Typically, only update one notebook per ticket branch, unless the ticket is to make similar updates to all notebooks
+(e.g., when bumping the RSP's recommended image).
+
+Update the README file when appropriate.
+
+Commit and push
+---------------
+
+Always restart the kernel and clear all outputs before saving, committing, and pushing changes.
+
+Pull request
+------------
+
+When the notebook is complete open a pull request to merge the ticket branch into the ``main`` branch (*not* to ``prod``). 
+
+Review
+------
+
+Contact one or more Rubin staff members (it need not be a Community Engagement team member) with the appropriate expertise
+and ask them to review the notebook.
+If they agree, assign them as a reviewer on your pull request.
+
+Ensure that all of the reviewers comments are addressed.
+Make changes and new commits to the branch, and respond to all of their comments with either a confirmation a change was made,
+or an explanation of why the request was not implemented. 
+
+Contact the reviewers to let them know the pull request now awaits their approval.
+
+Merge
+-----
+
+After the reviewers have approved the pull request, rebase and merge your ticket branch into the ``main`` branch (*not* to prod).
```

Comment:
In the HTML version, there is a space missing between "not" and "to prob".

Reply:
Prod should have different formatting: ``prod``

Reply:
I think there is a space between not and "to prod", also changed prod to ``prod``